scripts/monitoring/cron-send-build-app.py

```diff
@@ -0,0 +1,194 @@
+
+        if proj_name:
+            cmd.append('-n%s'%proj_name)
+
+        return json.loads(OpenShiftOC.oc_cmd(cmd, verbose))
+
+    @staticmethod
+    def new_app(path, proj_name, verbose=False):
+        '''run new-app
+        '''
+        cmd = ['new-app', path, '-n', proj_name]
+        return OpenShiftOC.oc_cmd(cmd, verbose)
+
+    @staticmethod
+    def delete_project(name, verbose=False):
+        '''delete project
+        '''
+        cmd = ['delete', 'project', name]
+        results = OpenShiftOC.oc_cmd(cmd, verbose)
+        time.sleep(5)
+        return results
+
+    @staticmethod
+    def new_project(name, verbose=False):
+        '''create new project
+        '''
+        cmd = ['project']
+        results = OpenShiftOC.oc_cmd(cmd, verbose)
+        curr_project = results.split()[2].strip('"')
+        cmd = ['new-project', name]
+        rval = OpenShiftOC.oc_cmd(cmd, verbose)
+        cmd = ['project', curr_project]
+        results = OpenShiftOC.oc_cmd(cmd, verbose)
+        return rval
+
+    @staticmethod
+    def get_projects(verbose=False):
+        '''get all projects
+        '''
+        cmd = ['get', 'projects', '--no-headers']
+        results = [proj.split()[0] for proj in OpenShiftOC.oc_cmd(cmd).split('\n') if proj and len(proj) > 0]
+        if verbose:
+            print results
+
+        return results
+
+    @staticmethod
+    def oc_cmd(cmd, verbose=False):
+        '''Base command for oc
+        '''
+        cmds = ['/usr/bin/oc']
+        cmds.extend(cmd)
+        if verbose:
+            print ' '.join(cmds)
+        proc = subprocess.Popen(cmds, stdout=subprocess.PIPE, stderr=subprocess.PIPE, \
+                                env={'KUBECONFIG': '/etc/origin/master/admin.kubeconfig'})
+        proc.wait()
+        if proc.returncode == 0:
+            output = proc.stdout.read()
+            if verbose:
+                print output
+                print
+            return output
+
+        return "Error: %s.  Return: %s" % (proc.returncode, proc.stderr.read())
+
+
+def curl(ip_addr, port):
+    ''' Open an http connection to the url and read
+    '''
+    return urllib.urlopen('http://%s:%s' % (ip_addr, port)).read()
+
+def main():
+    ''' Do the application creation
+    '''
+    proj_name = 'ops-monitor-appbuild' + os.environ['ZAGG_CLIENT_HOSTNAME']
+    app = 'nodejs-example'
+    verbose = False
+
+    if len(sys.argv) > 1 and sys.argv[1] == '-v':
+        verbose = True
+
+    start_time = time.time()
+    if proj_name in  OpenShiftOC.get_projects(verbose):
+        OpenShiftOC.delete_project(proj_name, verbose)
+
+    OpenShiftOC.new_project(proj_name, verbose)
+
+    OpenShiftOC.new_app(app, proj_name, verbose)
+    #1 is error
+    create_app = 1
+    # Now we wait until the pod comes up
+    for _ in range(24):
+        time.sleep(5)
+        #checking the building pod 
+        buildPod=OpenShiftOC.get_build_pod('nodejs-example',proj_name,verbose)
```

Comment:
This should be:
----
+        buildPod=OpenShiftOC.get_build_pod(app,proj_name,verbose)
----